src/states/infiniteweightpeps.jl

```diff
@@ -296,17 +324,212 @@ function InfiniteWeightPEPS(peps::InfinitePEPS)
+            |         |         |
+    --y₁₃---┼---y₂₃---┼---y₃₃---┼...y₁₃...
+            |         |         |                     2
+            x₁₂       x₂₂       x₃₂                   ↓
+            |         |         |                     x
+    --y₁₂---┼---y₂₂---┼---y₃₂---┼...y₁₂...            ↓
+            |         |         |                     1
+            x₁₁       x₂₁       x₃₁
+            |         |         |           2 -→ y -→ 1
+    --y₁₁---┼---y₂₁---┼---y₃₁---┼...y₁₁...
+            :         :         :
+            x₁₃       x₂₃       x₃₃
+            :         :         :
+    ```
+    We need to further:
+    - Move 1st column of x-weights to the last column.
+    - Permute axes of x-weights.
+    - Flip x-arrows from → to ←. 
+
+- After `rotr90`, x/y-weights are exchanged. 
+    ```
+            :         :         :
+            x₃₃       x₂₃       x₁₃
+            :         :         :
+    ..y₁₁...┼---y₃₁---┼---y₂₁---┼---y₁₁---
+            |         |         |           1 ←- y ←- 2
+            x₃₁       x₂₁       x₁₁
+            |         |         |           1
+    ..y₁₂...┼---y₃₂---┼---y₂₂---┼---y₁₂---  ↑
+            |         |         |           x
+            x₃₂       x₂₂       x₁₂         ↑
+            |         |         |           2
+    ..y₁₃...┼---y₃₃---┼---y₂₃---┼---y₁₃---
+            |         |         |
+            x₃₃       x₂₃       x₁₃
+            |         |         |
+    ```
+    We need to further:
+    - Move last row of y-weights to the 1st row. 
+    - Permute axes of y-weights. 
+    - Flip y-arrows from ↑ to ↓. 
+
+After `rot180`, x/y-weights are not exchanged. 
+    ```
+            :         :         :
+            y₁₃       y₁₂       y₁₁
+            :         :         :
+    --x₃₃---┼---x₃₂---┼---x₃₁---┼...x₃₃...
+            |         |         |           2 -→ x -→ 1
+            y₃₃       y₃₂       y₃₁
+            |         |         |                     1
+    --x₂₃---┼---x₂₂---┼---x₂₁---┼...x₂₃...            ↑
+            |         |         |                     y
+            y₂₃       y₂₂       y₂₁                   ↑
+            |         |         |                     2
+    --x₁₃---┼---x₁₂---┼---x₁₁---┼...x₁₃...
+            |         |         |
+            y₁₃       y₁₂       y₁₁
+            |         |         |
+    ```
+    We need to further:
+    - Move 1st column of x-weights to the last column.
+    - Move last row of y-weights to the 1st row.
+    - Permute axes of all weights and twist their axis 1. 
+    - Flip x-arrows from → to ←, and y-arrows from ↑ to ↓. 
+=#
+
+"""
+Mirror an `SUWeight` by its anti-diagonal line.
+"""
+function mirror_antidiag(wts::SUWeight)
+    weights2_x = mirror_antidiag(wts[2, :, :])
+    weights2_y = mirror_antidiag(wts[1, :, :])
+    return SUWeight(weights2_x, weights2_y)
+end
+function Base.rotl90(wts::SUWeight)
+    wts_x = rotl90(wts[2, :, :])
+    Nc = size(wts_x, 2)
+    wts_x = wts_x[:, vcat(2:Nc, 1)]
```

Comment:
```suggestion
    wts_x = circshift(rotl90(wts[2, :, :]), (0, 1))
```
I always forget the direction so I'm not sure if it should be +1 or -1 for the shift.

Reply:
Just checked: it is -1 in this case.
```julia
julia> a = collect(1:5);

julia> println(circshift(a, -1))
[2, 3, 4, 5, 1]
```